### PR TITLE
DOCS-18 Updated PCLI Reference guide #time 3h

### PR DESCRIPTION
**Pull request by MichaelBaj**
Updated:
  - show ospf
  - show ospf border-routers
  - show ospf database
  - show ospf database lsa
  - show ospf database max-age
  - show ospf interfaces
  - show ospf neighbors
  - show ospf routes
  - show peers
  - show platform
  - show rib
  - show rib bgp
  - show rib connected
  - show rib ospf
  - show rib static
  - show rib summary
  - show security key-status
  - show sessions
  - show sessions top bandwidth

### CODE REVIEW

File: docs/cli_reference.md [hunk 1]
```diff
@@ -2543,61 +2543,1056 @@ Available to _admin_ and _user_ accounts. Updated in 3.0, added _node_ keyword t
...
+
+Completed in 0.30 seconds
+```
+
+
+### show ospf routes
+
+#### Usage
+```
+show ospf neighbors [network-interface <name>] [neighbor <ip>] [force] [router <router>] [<verbosity>]
```
Comment (ptimmons) [inline docs/cli_reference.md:3401]: Copy/paste errors here.

File: docs/intro_installation.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The examples listed in this guide generally prefer running commands as a non-roo
 ## Hardware Requirements
 The 128T routing software runs on both bare metal servers and as a virtual machine within hypervisor environments. For virtual environments, the same CPU, memory, and storage specifications are required for comparable throughput.
 
-The 128T routing software recommends a minimum of two CPU cores, 8GB of RAM, and at least 25GB of hard drive space.
+The 128T routing software recommends a minimum of 4 CPU cores, 8GB of RAM, and at least 25GB of hard drive space.
```
Comment (ptimmons) [inline docs/intro_installation.md:21]: numbers below ten are usually written out. recommend "four CPU cores"

File: docs/cli_reference.md [hunk 3]
```diff
@@ -446,136 +570,135 @@ admin@cnd1.conductor#
...
 ```
 
-#### Description
+##### Keyword Arguments
+- **force**     Skip confirmation prompt
+- **node**      the node from which to delete flow entries
+- **router**    the router from which to delete flow entries
 
+#### Description
 The _delete flows_ command clears all active flow data from this node. Administrators can specify which node to clear flow data from by adding the node name as an optional argument to the command.
```
Comment (ptimmons) [inline docs/cli_reference.md:605]: This command has been maintained for backward compatibility to older versions of software. The _delete sessions_ command is preferred in versions newer than 3.2.0.

File: docs/cli_reference.md [hunk 4]
```diff
@@ -1270,222 +1522,639 @@ Configuration Attributes:
...
+- **tenant**          name of source tenant for ICMP request (default is the global tenant)
+- **timeout**         time to wait for a response, in seconds [max: 10 seconds] [type: int] (default: 1)
+
+##### Positional Arguments
+- **destination-ip**    destination IP of the ping request [type: IP address]
+
+#### Description
+This issues ICMP requests to the specified _destination-ip_, and offers the administrators a variety of ways to formulate the request. The _tenant_ and _service-name_ modifiers specify which "source tenant" to use for the request, and the name of the service for which the _destination-ip_ applies. The _count_ modifier will affect the number of pings that are issued. The _interface_ modifier lets administrators specify the egress interface for issuing the pings. The _timeout_ modifier will set the waiting period for a reply before declaring the ping as a failure. The _set-df-bit_ and _record-route_ options enable the respective flags in the outgoing ICMP request.
+
+#### Example
```
Comment (ptimmons) [inline docs/cli_reference.md:2029]: This example looks old, should be updated to use `service-ping`.

File: docs/cli_reference.md [hunk 5]
```diff
@@ -2275,155 +3017,473 @@ Completed in 0.18 seconds
...
+- **force**     Skip confirmation prompt
+- **name**      Network interface to display (if omitted, all will be displayed)
+- **node**      node for which to display dhcp lease info
+- **router**    router for which to display dhcp lease info
+
+##### Positional Arguments
+- **verbosity**    detail | summary (default: summary)
+
 #### Description
+Display dhcp lease info for network-interfaces
```
Comment (ptimmons) [inline docs/cli_reference.md:3125]: DHCP

File: docs/cli_reference.md [hunk 6]
```diff
@@ -2452,257 +3512,1517 @@ intf10.0 gateway 172.16.12.1
...
-show ntp [node <node-name>]
+show network-interface application [name <name>] [node <node>]
 ```
 
+##### Keyword Arguments
+- **name**      Network interface to display (if omitted, all will be displayed)
+- **node**      node for which to display network-interface data
+
 #### Description
+The command _show network-interface application_ can be used to display information regarding DHCP client certificates when running a DHCP server on the respective _network-interface_.
```
Comment (ptimmons) [inline docs/cli_reference.md:3583]: DHCP client reservations, not certificates

File: docs/cli_reference.md [hunk 7]
```diff
@@ -2452,257 +3512,1517 @@ intf10.0 gateway 172.16.12.1
...
+
+##### Subcommands
+- **hostnames**    Display resolved hostnames of peers
+
+#### Description
 The _show peers_ command displays properties of each of the "neighboring" 128T routers that the router in question has a peering association with.
 
 This command shows information on peering associations between 128T routers, not peering associations with BGP peers. For information on BGP peering statistics, refer to "show bgp" in this document.
 
 For each peer it shows which interface the peer is reachable via, the destination IP address for which the peer is reached, the VLAN to use to reach it, and whether the peer is currently "up", "down", or "initializing".
```
Comment (ptimmons) [inline docs/cli_reference.md:4655]: Worth mentioning:
The _detail_ option will show peer path statistics (loss, latency, jitter, calculated MOS, uptime) for each peer path.

File: docs/cli_reference.md [hunk 8]
```diff
@@ -5,71 +5,184 @@ sidebar_label: 'Command Line Reference'
...
 
-## clear arp
+#### Usage
+```
+clear context router
+```
 
-#### Syntax
+#### Description
+Clear both the router context and node context
```
[the remark this answers is not in the document]
Comment (MichaelBaj) [reply docs/cli_reference.md:86]: I am in favor of ending with punctuation as well.  These are mistakes on my part.

File: docs/cli_reference.md [hunk 9]
```diff
@@ -238,50 +360,63 @@ config
 The full suite of components, and relationships among those components, within the configuration tree are described in the Configuration Elements Reference section of this document.
```
[the remark this answers is not in the document]
Comment (MichaelBaj) [reply docs/cli_reference.md:360]: Removed this sentence.  The Configuration Elements document does not yet exist on the docs repo.  This language came from the original document.

**Pull request merged**